pkg/resource/scalable_target/testdata/test_suite.yaml

```diff
@@ -1,17 +1,115 @@
 tests:
-  - name: "Scalable target demo test"
-    description: "Scalable target CRD tests"
+  - name: "ScalableTarget create tests"
+    description: "ScalableTarget create tests"
     scenarios:
-     - name: "Put=InvalidInput"
-       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
+     - name: "Register=InvalidInput"
+       description: "Given one of the parameters is invalid, the status should show a error message."
        given:
-         desired_state: "scalable_target/v1alpha1/st_invalid_before_create.yaml"
+         desired_state: "scalable_target/v1alpha1/create/desired/invalid_before_create.yaml"
          svc_api:
            - operation: RegisterScalableTargetWithContext
              error:
                code: InvalidParameterValue
                message: "The service namespace must be one of the following: appstream | cassandra | comprehend | custom-resource | dynamodb | ec2 | ecs | elasticmapreduce | kafka | lambda | rds | sagemaker."
        invoke: Create
        expect:
-         latest_state: "scalable_target/v1alpha1/st_invalid_create_attempted.yaml"
+         latest_state: "scalable_target/v1alpha1/create/observed/invalid_create_attempted.yaml"
          error: "InvalidParameterValue: The service namespace must be one of the following: appstream | cassandra | comprehend | custom-resource | dynamodb | ec2 | ecs | elasticmapreduce | kafka | lambda | rds | sagemaker.\n\tstatus code: 0, request id: "
+     - name: "Register=ValidInput"
+       description: "Register a new ScalableTarget successfully."
+       given:
+         desired_state: "scalable_target/v1alpha1/create/desired/valid_before_create.yaml"
+         svc_api:
+           - operation: RegisterScalableTargetWithContext
+             output_fixture: "scalable_target/sdkapi/create_success.yaml"
+       invoke: Create
+       expect:
+         latest_state: "scalable_target/v1alpha1/create/observed/valid_created_no_role.yaml"
+         error: nil
+  - name: "ScalableTarget delete tests"
+    description: "ScalableTarget delete tests"
+    scenarios:
+      - name: "Deregister=Success"
```

Comment:
nit: convention is to name `<invoke_function>=<value>` 
in this case it will be `Delete=Success`